content/_data/events/gothenburg_jam_oct_2017

```diff
@@ -0,0 +1,6 @@
+---
+title: "Gothenburg JAM"
+date: "2017-10-11T6:00:00"
```

Comment:
This needs to be in 24-hour format, i.e. `18:00:00` rather than `6:00:00`.

See also #1177.